packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableRowAction.tsx

```diff
@@ -57,10 +57,29 @@ export function TransactionsTableRowAction({
   const { claim: claimCctp, isClaiming: isClaimingCctp } = useClaimCctp(tx)
   const { isConfirmed } = useRemainingTime(tx)
 
-  const { isEthereum, isArbitrumOne, isArbitrumGoerli } = isNetwork(chainId)
-  const currentChainIsValid =
-    (type === 'deposits' && (isArbitrumOne || isArbitrumGoerli)) ||
-    (type === 'withdrawals' && isEthereum)
+  const { isEthereum, isArbitrumOne, isArbitrum, isArbitrumGoerli } =
+    isNetwork(chainId)
+
+  const currentChainIsValid = useMemo(() => {
+    const isWithdrawalSourceOrbitChain = isNetwork(l2Network.id).isOrbitChain
+
+    if (isWithdrawalSourceOrbitChain) {
+      // Enable claim if withdrawn from an Orbit chain and is connected to L2
+      return isArbitrum
+    }
+
+    return (
+      (type === 'deposits' && (isArbitrumOne || isArbitrumGoerli)) ||
```

Comment:
We can use `isArbitrum` instead (or we need to add Sepolia)

Reply:
`isArbitrum` should work, no need to specify them one by one as it's already showing results based on selected chains

Reply:
updated